driver-async/src/main/com/mongodb/async/client/MongoCollection.java

```diff
@@ -1219,6 +1221,17 @@ void findOneAndUpdate(ClientSession clientSession, Bson filter, Bson update, Fin
      */
     void createIndexes(List<IndexModel> indexes, SingleResultCallback<List<String>> callback);
 
+    /**
+     * Create multiple indexes. If successful, the callback will be executed with a list of the namess of the created index as the result.
```

Comment:
Copy and paste of a spelling error and grammatical error.  Should be:

     ...with a list of the names of the created indexes...

Reply:
Done